user_guide/running_pipelines.qmd

```diff
@@ -1,37 +1,81 @@
+  -revision 0.12.1 \
+  -latest \
+  --publish_dir foo/ \
+  --input "bar" \
+  --output "test.txt" \
+  -c configs/my_hpc.config
 ```
 
-### Running pipelines from Nextflow Tower
+## Using the Nextflow Tower CLI
+
+If you have access to a Nextflow Tower instance in which a Compute Environment has already been set up, you can run a workflow from the Tower CLI. The command is very similar to the command to run a workflow from the CLI, but you need to:
+  * Use `tw launch` instead of `nextflow run`
+  * Specify the workspace ID and compute environment ID
+  * Rename arguments: `-revision` to `--revision`, `-latest` to `--pull-latest`, `-main-script` to `--main-script`, `-c` to `--config`
+  * Store workflow arguments in a separate yaml file (if this was not already the case).
+
+Example:
+
+```bash
+tw launch openpipelines-bio/openpipeline \
+  --revision 0.12.1 \
+  --pull-latest \
+  --main-script target/nextflow/workflows/integration/multimodal_integration/main.nf \
+  --workspace <your workspace id> \
+  --compute-env <your compute environment id> \
+  --params-file params.yaml \
+  --config configs/my_hpc.config
+```
+
+## Using the Nextflow Tower Web UI
+
+If you have access to a Nextflow Tower instance in which a Compute Environment has already been set up, you can run a workflow from the Tower UI. To do so, go to the "Launchpad" and click on the button "launch a run without configuration".
+
+![](./images/launchpad.png)
+
+Next, fill in the required fields and click on "Launch run".
+
+* **Compute environment**: Select the compute environment you want to run the workflow on.
+* **Pipeline to launch**: Fill in `openpipelines-bio/openpipeline`.
+* **Revision number**: The release number of the pipeline you want to run, e.g. `0.12.1`. You can find the release number on the [GitHub releases page](https://github.com/openpipelines-bio/openpipeline/releases).
+* **Work directory**: The bucket path where the scratch data is stored.
+* **Pipeline parameters**: The YAML or JSON of the parameters that are passed to the pipeline. See the [Components](../components/) page for more information about the parameters of each pipeline.
 
-### Using param_list to pass large parameter sets
+## Using param_list to pass large parameter sets
 
-Using Viash's VDSL3 nextflow platform, an optional `-param_list` argument can be used to pass a large number of inputs to a workflow. Additionally, the pipeline parameter values can be set for each input to the workflow independently. A `param_list` can either be a list of maps, a csv file, a json file, a yaml file, or simply a yaml blob:
+Using Viash's VDSL3 nextflow platform, an optional `--param_list` argument can be used to pass a large number of inputs to a workflow. Additionally, the pipeline parameter values can be set for each input to the workflow independently. A `param_list` can either be a list of maps, a csv file, a json file, a yaml file, or simply a yaml blob:
```

Comment:
Is it intentional that this information is now duplicated?

Reply:
Nope! It used to be in the same page but since there was quite a lot of info in the new page, I wanted to split it up.

Reply:
Fixed ;)